src/fetch_usa_bis.ts

```diff
@@ -0,0 +1,39 @@
+const parse = require('csv-parser')
+const https = require('https')
+
+let res: any = [];
+let path: string = 'https://www.bis.doc.gov/dpl/dpl.txt';
+
+function parseCSV() {
+  let keys: string[] = [];
+  let values: any;
+  return new Promise((resolve: any, reject: any) => {
+    https.get(path, (stream: any) => {
+      stream.pipe(parse())
+      .on('data', function(csvrow: any) {
+        let newrow: any = {};
+        keys = Object.keys(csvrow)[0].replaceAll('"', '').split('\t')
```

Comment:
Not sure why removing "? 
Can you explain what this line does in a code comment?

Reply:
May not remove ",  But if not remove, will make Object like this.
 {
    '"Street_Address"': '"UPON THE DATE OF THE ORDER INCARCERATED AT USM NO: 29686-177, FCI SEAGOVILLE, 2113 NORTH HIGHWAY 175"',    
    '"City"': '"SEAGOVILLE"',
    '"State"': '"TX"',
    '"Country"': '"US"',
    ...
  },